### PR TITLE
[Relay] Fix Fuse

### PR DESCRIPTION
**Pull request by MarisaKirisame**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

**Comment by MarisaKirisame** [comment]
@tqchen @masahi this fix #3026 .

File: src/relay/pass/fuse_ops.cc [hunk 1]
```diff
@@ -865,9 +865,17 @@ class FuseMutator : private ExprMutator {
   }
 
   Expr MakeNewFunction(GraphPartitioner::Group* group, Type ret_type, Expr body) {
+    // If the function has no call, it is not a primitive function.
+    struct HasCallVisitor : ExprVisitor {
+      bool HasCall = false;
```
Comment (masahi) [inline src/relay/pass/fuse_ops.cc]: should be snake case

**Comment by tqchen** [comment]
I believe that we will need a bit more discussion about what is happening here. In particular, the usage of fusor and primitive functions. There might be a better fix to this problem e.g. (not fuse these ops)

**Comment by MarisaKirisame** [comment]
@tqchen I believe the fusion algorithm need some update to handle tuple correctly - see #2931 . I think we should chat a bit as I am confused about the code. In particluar sometimes information flow forward and sometimes backward (but in dataflow analysis it is always one directional AFAICT).

**Comment by tqchen** [comment]
Move discussion to https://github.com/dmlc/tvm/issues/3039

**Pull request merged**